src/actions/query.js

```diff
@@ -172,6 +172,10 @@ function msearch(
 				});
 			}
 
+			if (config.transformResponse && typeof config.transformResponse) {
+				config.transformResponse(res);
```

Comment:
Shouldn't it be:

```suggestion
 				res = config.transformResponse(res);
```

Reply:
In that case, we might have to `await` for `tranfromResponse`,
Because we might have to wait for the response from different data source like `graphql-server`
I think it should be more like `res = await config.transformResponse(res)`

Reply:
Yeah, that makes sense. We can use a promise based spec for `transformResponse`. 💯

Reply:
👍 Have updated the PR accordingly